tests/integration/fixtures/relay.py

```diff
@@ -44,7 +46,12 @@ def shutdown(self, sig=signal.SIGKILL):
 @pytest.fixture
 def relay(mini_sentry, random_port, background_process, config_dir):
     def inner(
-        upstream, options=None, prepare=None, external=None, wait_healthcheck=True
+        upstream,
+        options=None,
+        prepare=None,
+        external=None,
+        wait_healthcheck=True,
+        static_relays_config=None,
```

Comment:
Let's remove reundant suffices.

```suggestion
        static_relays=None,
```

Reply:
Since we are constructing a relay object and passing a mini_sentry, I wanted to make it clear that we don't pass relay objects but configs, i think that in this case the naming is not redundant but I can change it if you prefer.